runtime/cmake/caches/aix_ppc-64.cmake

```diff
@@ -20,25 +20,34 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ################################################################################
 
-
-#TODO Platform hacks
+#TODO: Env vars should be auto detected by platform
 set(J9VM_ARCH_POWER ON CACHE BOOL "")
 set(J9VM_ENV_DATA64 ON CACHE BOOL "")
+set(J9VM_ENV_DLPAR ON CACHE BOOL "")
 set(J9VM_ENV_HAS_FPU ON CACHE BOOL "")
 set(J9VM_ENV_SHARED_LIBS_CALLEE_GLOBAL_TABLE_SETUP OFF CACHE BOOL "")
 set(J9VM_ENV_SHARED_LIBS_USE_GLOBAL_TABLE ON CACHE BOOL "")
 
-set(OMR_GC_TLH_PREFETCH_FTA OFF CACHE BOOL "")
-set(J9VM_JIT_RUNTIME_INSTRUMENTATION ON CACHE BOOL "")
-set(J9VM_PORT_RUNTIME_INSTRUMENTATION ON CACHE BOOL "")
-set(J9VM_MODULE_CODEGEN_PPC ON CACHE BOOL "")
 set(J9VM_GC_IDLE_HEAP_MANAGER OFF CACHE BOOL "")
+set(J9VM_GC_TLH_PREFETCH_FTA OFF CACHE BOOL "")
+set(J9VM_GC_SUBPOOLS_ALIAS ON CACHE BOOL "")
 set(J9VM_INTERP_ATOMIC_FREE_JNI ON CACHE BOOL "")
 set(J9VM_INTERP_ATOMIC_FREE_JNI_USES_FLUSH ON CACHE BOOL "")
 set(J9VM_INTERP_TWO_PASS_EXCLUSIVE ON CACHE BOOL "")
+set(J9VM_JIT_RUNTIME_INSTRUMENTATION ON CACHE BOOL "")
+set(J9VM_MODULE_CODEGEN_PPC ON CACHE BOOL "")
 set(J9VM_OPT_SWITCH_STACKS_FOR_SIGNAL_HANDLER OFF CACHE BOOL "")
+set(J9VM_PORT_RUNTIME_INSTRUMENTATION ON CACHE BOOL "")
 set(J9VM_THR_ASYNC_NAME_UPDATE OFF CACHE BOOL "")
 
+
+set(OMR_GC_CONCURRENT_SCAVENGER ON CACHE BOOL "")
+set(OMR_GC_HEAP_CARD_TABLE ON CACHE BOOL "")
+set(OMR_PORT_NUMA_SUPPORT ON CACHE BOOL "")
```

Comment:
Where is this enabled for AIX outside of cmake?

Reply:
hmm. not sure how that made it in here. Removed